contractsSol5/Dao/KyberDAO.sol

```diff
@@ -655,8 +621,8 @@ contract KyberDAO is IKyberDAO, EpochUtils, ReentrancyGuard, CampPermissionGroup
     }
 
     /**
-    * @dev helper func to get encoded reward and rebate
-    * @dev revert if validation failed
+    * @dev  helper func to get encoded reward and rebate
+    *       revert if validation failed
```

Comment:
is function still required?

Reply:
it is used to get encoded data from reward & rebate, useful for creating new BRR campaign